### PR TITLE
add visual regression tests

### LINKED ISSUE #435: Add visual regression tests using Cypress for static pages
# Feature request
For pages that are _NOT_ `code_schools`, `login`, `join`, and `profile`, we can improve our shipping confidence with [visual regression tests](https://www.abhishek-tiwari.com/visual-regression/). 

**Describe the solution you'd like**
Cypress will [eventually ship 1st class support](https://github.com/cypress-io/cypress/issues/495), but in the meantime, [solutions exist that leverage only one external library](https://medium.com/norwich-node-user-group/visual-regression-testing-with-cypress-io-and-cypress-image-snapshot-99c520ccc595).

### PR DESCRIPTION
**Pull request by markchernov**
# Description of changes

Adds static pages visual regression tests with Cypress 
# Issue Resolved

Fixes #435

### CODE REVIEW

**Comment by kylemh** [comment]
🤖 **This is a bot** 🤖<br /><br />🎉 Deployed Storybook preview! 🎉<br /><br />Click the link at the bottom of this comment to see it live.

Built with commit e61ee7ce7af7dbdba996c6832ba7afd61cf6e569

https://deploy-preview-487--operation-code-storybook.netlify.com

File: cypress.json [hunk 1]
```diff
@@ -3,5 +3,6 @@
   "baseUrl": "http://localhost:3000",
   "viewportWidth": 1600,
   "viewportHeight": 1200,
-  "requestTimeout": 10000
+  "requestTimeout": 10000,
+  "video": false
```
Comment (kylemh) [inline cypress.json]: What does this do?

Comment (markchernov) [reply cypress.json]: I think it stops the video recording of a test run. I noticed Travis would create mpeg4 after each test. Not sure if somebody looks at those.

Comment (kylemh) [reply cypress.json]: Mmmm yeah I want this reactivated. I’ll do it in another PR

**Pull request merged**